mlflow/tracing/config.py

```diff
@@ -0,0 +1,112 @@
+from copy import deepcopy
+from dataclasses import dataclass
+from typing import Callable, Optional
+
+from mlflow.tracing.utils.processor import validate_span_processor
```

Comment:
```suggestion
from mlflow.tracing.utils.processor import validate_span_processors
```
since validate_span_processor is an alias, we might want to clean it up later?